src/components/TestRail/TestRail.js

```diff
@@ -174,42 +182,84 @@ class TestRail {
      *
      * @param {string} runID
      * @param {Result[]} testResults
+     * @param {boolean} includeCasesOnlyIfExistInRun
      * @returns {Promise<AxiosResponse<*>>}
      */
-    sendBatchResults(runID, testResults) {
-        const url = '/add_results_for_cases/' + runID;
-
-        const postData = {
-            results: [],
-        };
+    async sendBatchResults(runID, testResults, includeCasesOnlyIfExistInRun) {
```

Comment:
change this to match
```suggestion
    async sendBatchResults(runID, testResults, ignoreMissingCaseIds) {
```